### ISSUE TITLE
FSE: CoBlocks Hero Block in Template throws error if settings sidebar open [3]

### ISSUE THREAD

**Issue by shaunandrews** [Automattic/wp-calypso#34893]
We have a demo site setup for the FSE plugin at fsedemo.wordpress.com. I noticed that there was an extra Hero block (from Coblocks) in the Footer. I attempted to edit the footer and remove this block, but I found that anytime I try to focus on the Hero block Gutenberg crashes.

![image (4)](https://user-images.githubusercontent.com/191598/61894831-39130a80-aedf-11e9-8fd7-a936e920e1e9.png)

To Reproduce:
- Add a `full-site-editing` site sticker to your Simple Site
- Select the ~~Modern Business~~ Maywood theme
- Edit the header or footer template and add a Hero Block
- NOTE: you **must** have the settings sidebar open for the error to appear.

Note from @gwwar: If we end up needing to fork the CoBlocks Library, please remember that serialized block output should not diverge, and that we should file an upstream PR if it makes sense. If output changes folks will run into trouble on Atomic sites with the full plugin installed.

## Update:
It also looks like don't see the block rendering in the page editor

To Reproduce:
1. Follow the steps above, but don't have the settings sidebar open and you'll see the hero block in the template editor just fine.
2. go back to the page editor. It will not render there even though it does on the front end.

**Comment by shaunandrews** [comment]
> “TypeError: Cannot read property ‘_coblocks_dimensions’ of undefined”

**Comment by glendaviesnz** [comment]
At a quick look it seems like it is an issue with our template parts not saving post meta data - CoBlocks assigns _coblocks_dimensions  to post meta, and this works as expected in the post content, but post meta is undefined in the template parts by the looks.

**Comment by noahtallen** [comment]
Looks like this issue has changed a bit in recent updates. Instead of throwing an error, I just don't see it in the page editor:

## Template Editor:
<img width="1896" alt="Screen Shot 2019-08-15 at 1 15 06 PM" src="https://user-images.githubusercontent.com/6265975/63123825-c08fee80-bf5e-11e9-8e0e-6eef7aaa4d4d.png">


## Page Editor:
<img width="1902" alt="Screen Shot 2019-08-15 at 1 13 31 PM" src="https://user-images.githubusercontent.com/6265975/63123795-ae15b500-bf5e-11e9-8402-b6cd59606dec.png">

## Front End:
<img width="1846" alt="Screen Shot 2019-08-15 at 1 13 18 PM" src="https://user-images.githubusercontent.com/6265975/63123794-ae15b500-bf5e-11e9-8247-8869212478e8.png">

**Comment by mmtr** [comment]
Confirmed this is a side effect of https://github.com/godaddy/coblocks/issues/750. If the block is added typing the `/hero` shortcut, the error does not happen.

**Comment by kwight** [comment]
Looks like [a fix](https://github.com/godaddy/coblocks/pull/809) for this is upcoming in 1.13.0, which has a release date of [today](https://github.com/godaddy/coblocks/milestone/12).

**Comment by kwight** [comment]
This is still a problem with 1.13.0. While https://github.com/godaddy/coblocks/issues/750 now allows us to use either method for adding a Hero block to a post content section, attempting to add one to an FSE header or footer produces the same crash (and the same `TypeError: Cannot read property ‘_coblocks_dimensions’ of undefined` console error).

**Comment by kwight** [comment]
This is still valid, with the same error: `TypeError: Cannot read property '_coblocks_dimensions' of undefined`.

**Comment by gwwar** [comment]
Is there still behavior difference when using the keyboard inserter vs main inserter?

**Comment by Copons** [comment]
Ah! We were tracking this in https://github.com/Automattic/wp-calypso/issues/36888

> The error is thrown by this CoBlocks line, because `meta` is undefined.
> 
> https://github.com/godaddy/coblocks/blob/e136ef29c3e8c1f77199c207791f8692be2e405a/src/components/dimensions-control/index.js#L184
> 
> It has been fixed in [godaddy/coblocks#726](https://github.com/godaddy/coblocks/pull/726) and released in version 1.16.
> We are currently on 1.15 in prod and 1.16.1 in horizon.
> I've tried loading the 1.16.1 on my sandbox, and it fixed the issue! 🥂
> 
> I'll close this since we can't do anything about it except wait for the upgrade.

tl;dr
this has been fixed in 1.16 (you can test it in WPCOM right now by loading the 1.16.1 version) 🙂
